marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/entities/entity-search-lib.sjs

```diff
@@ -637,12 +638,30 @@ function findFlowsAsMap() {
   return flows;
 }
 
+function getRecordSize(docUri) {
```

Comment:
Should be "getDocumentSize", since "record" refers to the abstraction containing the URI, the document, metadata, etc - we're only interested in the document size.